### ISSUE TITLE
Disconnect and then connect again!

### ISSUE THREAD

**Issue by lucktu** [ntop/n2n#658]
edge1 -a 10.10.10.10 -c n2n -k test -l t1.lucktu.com:10082 -b
edge2 -a 10.20.10.10 -c n2n -k test -l t1.lucktu.com:10086
edges -a 10.21.10.10 -c n2n -k test -l t1.lucktu.com:10088 -b
edge3 -a 10.30.10.10 -c n2n -k test -l t1.lucktu.com:10090

These are my test connections(ping 10.30.10.10), and in the same way that v1\v2\v2s doesn't break, V2.9 does. It happens all the time (windows ping ubuntu, or ubuntu ping ubuntu).
![image](https://github.com/lucktu/other/raw/master/image/2021/21030601.png)

**Comment by Logan007** [comment]
This reminds me of #519 for which I developed #645 to counter intermediate blocks that could have been caused by steadily re-occuring name resolution. #645 is not merged yet because CI Travis does not digest it (no idea why). But it compiles well on my system. Perhaps it works for you? The code can be obtained from github's [archive](https://github.com/Logan007/n2n/archive/threadedRes.zip).

Other than that, I do not have to ask you if you are using the most current dev, right? :wink:

Unfortunately, I do not have seen such symptoms over here. I will further try to trigger such behavior, also using your parameters. As soon as I can reliably re-produce it, I will start a specific and never-fading bug hunt – I promise!
